primitives/src/lib.rs

```diff
@@ -539,8 +539,8 @@ impl Asset {
 #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "std", serde(rename_all = "camelCase"))]
 pub enum CurrencyId {
-	XCM(ForeignCurrencyId),
 	Native,
+	XCM(ForeignCurrencyId),
 	Stellar(Asset),
```

Comment:
Can you now add explicit numbers here? ie
```
Native = 0,
XCM(ForeignCurrencyId) = 1,
Stellar(Asset) = 2,
```

Let's also add numbers to the `Asset` enum just to be safe.

Reply:
ah, can't probably assign a number for non "unit" variant

Reply:
i can only set "0" for the first ones